db/db_utils.go

```diff
@@ -199,6 +199,24 @@ func DropAllIndexes(collections ...string) error {
 	return nil
 }
 
+// DropDatabases drops all of the given databases, returning an error immediately
+// if dropping any of the databases fails.
+func DropDatabases(dbs ...string) error {
+	session, _, err := GetGlobalSessionFactory().GetSession()
+	if err != nil {
+		return err
+	}
+	defer session.Close()
+
+	for _, db := range dbs {
+		if err := session.DB(db).DropDatabase(); err != nil {
+			return errors.Wrapf(err, "dropping database '%s'", db)
```

Comment:
`DB` ?

Reply:
Eh, either one works since DB is just an acronym for database. I'm inclined to just leave it.